orchestration/dagster_orchestration/Dockerfile

```diff
@@ -1,31 +1,31 @@
-FROM python:3.7.8-slim
+## Builds the dagster "user code deployment" image that will
+## hold our pipelines/solids/etc.
 
-ARG DAGSTER_VERSION
+FROM python:3.9.1-slim as base
+
+ENV PYTHONFAULTHANDLER=1 \
+  PYTHONUNBUFFERED=1 \
+  PYTHONHASHSEED=random \
+  PIP_NO_CACHE_DIR=off \
+  PIP_DISABLE_PIP_VERSION_CHECK=on \
+  PIP_DEFAULT_TIMEOUT=100 \
+  POETRY_VERSION=1.1.5
```

Comment:
I'd like to figure out a way to keep this and the poetry version we've setup in the pyproject.toml file, but this should be ok for now.